tests/import_test.go

```diff
@@ -427,9 +427,9 @@ var _ = Describe("Namespace with quota", func() {
 	})
 
 	It("Should fail to create import pod in namespace with quota, with resource limits higher in CDIConfig", func() {
-		err := f.UpdateCdiConfigResourceLimits(int64(2), int64(1024*1024*1024), int64(2), int64(1*1024*1024*1024))
+		err := f.UpdateCdiConfigResourceLimits(int64(2), int64(512*1024*1024), int64(2), int64(512*1024*1024))
```

Comment:
Can you add to the commit message why the limits were changed?

Reply:
I did: Fixed some tests that were failing due to low resource availability in CRC. Basically the node didn't have the guarantees available.